Modules/EMCAL/src/RawTask.cxx

```diff
@@ -281,7 +287,8 @@ void RawTask::monitorData(o2::framework::ProcessingContext& ctx)
           first = false;
         }
         if (feeID > 40)
```

Comment:
we advise to "use curly braces even when the body of the statement fits on one line". I did not catch it before. I will approve this PR but you can keep it in your mind.